mtp_api/apps/core/management/commands/load_test_data.py

```diff
@@ -32,8 +36,8 @@ class Command(BaseCommand):
     help = textwrap.dedent(__doc__).strip()
 
     def add_arguments(self, parser):
-        parser.add_argument('--protect-superusers', action='store_true',
-                            help='Prevents superusers from being deleted')
+        parser.add_argument('--no-protect-superusers', action='store_true',
```

Comment:
there are references to `protect_superusers` in other places in the code, e.g.
https://github.com/ministryofjustice/money-to-prisoners-api/blob/master/mtp_api/apps/core/views.py#L203

would be safer to keep the variable name but swap the default by doing something like:
```
parser.add_argument('--no-protect-superusers', action='store_false', dest='protect_superusers' …
```

Reply:
Sorry only just saw this comment. I've updated those other references now to be consistent with this negation for consistency. Let me know if you're happy with this, or if you still want me to go with what you describe above